applications/StructuralMechanicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -25,17 +25,36 @@
 
 //Utilities
 #include "custom_utilities/sprism_neighbours.hpp"
+#include "custom_utilities/eigenvector_to_solution_step_variable_transfer_utility.hpp"
```

Comment:
this does not exist any more, your local changes overwrite the ones in master (actually everything you add in this file is removed in master)

Reply:
Ok I see it . My bad I will correct this in my next commit.